web/src/components/storage/space-policy-utils.jsx

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (c) [2023] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React, { useState } from "react";
+
+import { _, n_ } from "~/i18n";
+import { sprintf } from "sprintf-js";
+import { noop } from "~/utils";
+import { Button, ExpandableSection, Hint, HintBody } from "@patternfly/react-core";
+import { DeviceList } from "~/components/storage";
+
+const ListBox = ({ children, ...props }) => <ul role="listbox" {...props}>{children}</ul>;
+
+const ListBoxItem = ({ isSelected, children, onClick, ...props }) => {
+  if (isSelected) props['aria-selected'] = true;
+
+  return (
+    <li
+      role="option"
+      onClick={onClick}
+      { ...props }
+    >
+      {children}
+    </li>
+  );
+};
+
+/**
+ * Content for a space policy item
+ * @component
+ *
+ * @param {Object} props
+ * @param {Locale} props.locale
+ */
+const PolicyItem = ({ policy }) => {
+  const Title = () => {
```

Comment:
NP: `Title` and `Description` could be helper functions instead of components returning a string.

Reply:
NP: I told Áncor just the contrary :)